docs(update): task worker metrics config (#1114)
camunda/camunda-docs-manual: content/update/minor/716-to-717/jboss.md

---

> title: "Update a JBoss/Wildfly Installation from 7.16 to 7.17"

Reviewer: ```suggestion
title: "Update a JBoss EAP 6 or Wildfly / JBoss EAP 7 Installation from 7.16 to 7.17"
```

menu:
  main:

>     name: "JBoss AS/Wildfly"

Reviewer: ```suggestion
    name: "JBoss EAP 6 or Wildfly / JBoss EAP 7"
```

    identifier: "migration-guide-717-jboss"
    parent: "migration-guide-717"

---

> The following steps describe how to update the Camunda artifacts on a JBoss AS
> 7 and Wildfly server in a shared process engine scenario. For the entire

Reviewer: ```suggestion
The following steps describe how to update the Camunda artifacts on a JBoss EAP 6 or Wildfly / JBoss EAP 7 in a shared process engine scenario. For the entire
```

## Update REST API

The following steps are required to update the Camunda REST API on a JBoss/Wildfly instance:

1. Undeploy an existing web application with a name like `camunda-engine-rest`
2. Download the REST API web application archive from our [Maven Nexus Server][engine-rest]. Alternatively, switch to the private repository for
   the enterprise version (credentials from license required). Choose the correct version named `$PLATFORM_VERSION/camunda-engine-rest-$PLATFORM_VERSION-$CLASSIFIER.war`.
3. Deploy the web application archive to your JBoss/Wildfly instance.

## Update Cockpit, Tasklist, and Admin

The following steps are required to update the Camunda web applications Cockpit, Tasklist, and Admin on a JBoss/Wildfly instance:

1. Undeploy an existing web application with a name like `camunda-webapp`
2. Download the Camunda web application archive from our [Maven Nexus Server][webapp-jboss].
   Alternatively, switch to the private repository for the enterprise version (credentials from license required).
   Choose the correct version named `$PLATFORM_VERSION/camunda-webapp-jboss.war`.
3. Deploy the web application archive to your JBoss/Wildfly instance.

> [update-guide]: {{< ref "/update/minor/715-to-716/_index.md" >}}

Reviewer: ```suggestion
[update-guide]: {{< ref "/update/minor/716-to-717/_index.md" >}}
```
